### PR TITLE
Enable static volume binding in Prowlar chart

### PR DESCRIPTION
**Pull request by jlpedrosa**
**Description**
This PR adds the option to specify `volumeName` in the PVC of Prowlarr, this allows use existing volumes in the PVC,

# Linked issue(s)

Fixes #0000 | Relates #0000

**Checklist**

- [ ] Unit tests updated
- [ ] End user documentation updated

### Community Note

- Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

### CODE REVIEW

File: charts/prowlarr/values.yaml [hunk 1]
```diff
@@ -68,6 +68,9 @@ config:
       - ReadWriteOnce
     # -- Config name
     name: ""
+    # -- Name of the permanent volume to reference in the claim. Can be used
+    # -- bind to existing volumes
```
Comment (M0NsTeRRR) [inline charts/prowlarr/values.yaml:72]:
```suggestion
    # -- Name of the permanent volume to reference in the claim.
    # Can be used to bind to existing volumes.
```

**Comment by M0NsTeRRR** [comment]
Hello,

Thanks for you contribution.
You need to bump the helm chart following semver (0.3.0 in this case) and update the readme with `helm-docs` command.

Otherwise LGTM

**Comment by jlpedrosa** [comment]
@M0NsTeRRR  Updated the chart version and re-run helm docs! Thanks for your work and for the super quick review.

BTW, I managed to convince the jellyfin people to create an "official" helm chart: https://github.com/jellyfin/jellyfin-helm/

**Comment by M0NsTeRRR** [comment]
> @M0NsTeRRR Updated the chart version and re-run helm docs! Thanks for your work and for the super quick review.
> 
> BTW, I managed to convince the jellyfin people to create an "official" helm chart: https://github.com/jellyfin/jellyfin-helm/

Oh nice, I will subscribe to the issue and jump into if no one took the issue. Would be the best to have an official helm chart !

**Pull request merged**